### DISCUSSION TITLE
Can Milvus Use Azure Blob Storage Without Azure Kubernetes Service (AKS)?

### DISCUSSION THREAD

**Discussion by Ald392** [milvus-io/milvus#34979] [Q&A and General discussion] [answered]
Is it necessary to use Azure Kubernetes Service (AKS) for deploying Milvus if my goal is to use Azure Blob Storage as an external object storage ?

I wanted to use Azure Blob Storage in my standalone milvus deployment.

**Comment by xiaofan-luan** [comment, marked as answer]
No, it is not a must.
But. it is recommened to run milvus on a K8s cluster, not necessary to be AKS, can your host service
